### ISSUE TITLE
frequent USB disconnects with Version 13.12 

### ISSUE THREAD

**Issue by jstrebel** [daid/LegacyCura#671]
Hi, i upgraded to 13.12 and experienced frequent disconnets on the USB connection. I don't have any issue unsing Pronterface. USB communication runs stable.
PC running W7
3D Printer: Arduino Mega 2560 R4/ RAMPS 1.4 Firmware marlin 1.0.0

PS: After downgrading back to 13.12.2 the issue has gone. 
How can I help to provide more data to find the cause?
Best regards Jakob

**Comment by daid** [comment]
Odd, nothing has changed in USB printing from 13.10 compared to 13.12...

**Comment by jstrebel** [comment]
Daid, how can I help to find the root cause. I tried as well from a  MAC OSX 10.9.1 
Here I have disconnects with version 13.11.2 and 13.12. But using Pronterface I don't experience any Problem with the connection. This kind of behavior smells to me like a timing issue. Is CURA doing some retries before it goes into the disconnect status? Jakob

**Comment by daid** [comment]
Yes, Cura does some re-tries. But I have not looked at that code for a while. Does it go into a disconnected or an error status? The error status should give an "error log" button.

**Comment by shape667** [comment]
Using the new firmware in version 13.12 causes my system to turn off the extruder motor after about 1 hour of printing. (tried three times).
The old firmware keeps printing to the end. 
hope this helps to find a solution.

**Comment by jstrebel** [comment]
Daid, did you get the error log i have sent by mail?

**Comment by jstrebel** [comment]
Update: It looks the Issue I am facing is present in 13.11.2 and 13.12 a
If I set M143 S260 the problem remains.
Q: Why Pronterface does not disconnect?
![cura_temp_error](https://f.cloud.github.com/assets/2244311/1840962/5a4756c2-749b-11e3-9fa6-53b79fb13da0.png)

Here is the tail of the error Log:
Send: M105 T0
Recv: ok T:245.0 /249.0 B:20.1 /0.0 @:117 B@:0
Send: M105 T0
Recv: ok T:246.7 /249.0 B:20.1 /0.0 @:90 B@:0
Send: M105 T0
Recv: ok T:248.3 /249.0 B:20.1 /0.0 @:64 B@:0
Send: M105 T0
Recv: ok T:249.2 /249.0 B:20.1 /0.0 @:66 B@:0
Send: M105 T0
Recv: ok T:250.0 /249.0 B:20.1 /0.0 @:68 B@:0
Recv: Error:0
Recv: : Extruder switched off. MAXTEMP triggered !
Changing monitoring state from 'Operational' to 'Error: 0: Extruder switched...'
Recv: Error:Printer stopped due to errors. Fix the error and use M999 to restart. (Temperature is reset. Set it after restarting)

**Comment by jstrebel** [comment]
Update:May be this helps other people 
First: it is NOT a CURA Issue.
 The confusing thing to me was that with Pronterface the MAXTEMP was not triggered. (comparing the temperature heating curve of Pronterface and CURA showed me that Pronterface reached the hotend temperature slowly, without overshoot vs. CURA exceeds the preset hotend temperature. Finally I found that the MAXTEMP in my MARLIN was preset to 250. For not known reason the command M143 S260 did not work. By changing HEATHER_0_MAXTEMP to 260 in Configuration.h the Problem was fixed.)
